test/dotnet/Integration/RedisListTriggerTestFunctions.cs

```diff
@@ -41,6 +42,30 @@ public static class RedisListTriggerTestFunctions
             logger.LogInformation(IntegrationTestHelpers.GetLogValue(entry));
         }
 
+        [FunctionName(nameof(ListTrigger_Batch_RedisValue))]
+        public static void ListTrigger_Batch_RedisValue(
+            [RedisListTrigger(localhostSetting, nameof(ListTrigger_Batch_RedisValue), pollingIntervalInMs: pollingIntervalShort, count: count)] RedisValue[] entry,
```

Comment:
Not really related to batching but I'll mention it anyway - 
I'm not familiar with this pollingInterval setting but typically this type of setting lives at the host.json level instead of the function level, for example maxPollingInterval for storage queues:
https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-queue?tabs=in-process%2Cextensionv5%2Cextensionv3&pivots=programming-language-csharp#host-json

Reply:
Thanks for the heads up. It seems like that's the standard for other extensions, so I'll create a separate PR to move the polling interval setting to the host.json level instead of the function level.